### ISSUE TITLE
Feature request: command to get path to selected runtime

### ISSUE THREAD

**Issue by delicb** [jdxcode/rtx#20]
It would be great to be able to obtain path to single runtime as command, same value that is being injected to `$PATH`. 

My use case is that I want to set `pipx` to use currently selected Python. With `asdf` in Fish, I use `set -gx PIPX_DEFAULT_PYTHON "$HOME/.asdf/shims/python"`, but since `rtx` does not use shims, there is no "stable" path to use here. 

I imagine solution like:
`set -gx PIPX_DEFAULT_PYTHON (rtx runtime-path python)/python`

where `rtx runtime-path python` (or whatever command name, I don't have a preference here) would return the same path as is injected to `$PATH` (e.g. `$HOME/.local/share/rtx/installs/python/3.11.1/bin`). 

If there is other solution for this use case, I have not figured it out and would like to hear about it. 

btw, thanks for a great tool, creating and open-sourcing it, looking forward to using it actively :)

**Comment by jdxcode** [comment]
I technically _could_ add this (the data exists already) but every language I've seen has static runtime directories. They can override them with `bin/list-bin-paths` but most don't (including python). If a plugin doesn't have one then the paths are just `$INSTALL_DIR/bin`.

You should be able to do this with:

```
set -gx PIPX_DEFAULT_PYTHON (rtx where python)/bin/python
```

However I just noticed that `rtx where` isn't working correctly. `rtx where python` should give you the _current_ python version but it's giving the latest. I'll fix that shortly.

**Comment by delicb** [comment]
Thanks for the quick response.

I tried `rtx where`, but I stumbled on the problem you mentioned (in retrospect, I did not figure out that it is the problem) - it returned python `3.11.1` and I have `3.10.6` in `.tool-versions` (and I did grep, so it did not find it). With `rtx where` returning right answer - that solves my use case, I don't expect to have the need for plugin that override `bin/list-bin-paths`. 

Thanks, I will be watching releases for `rtx where` fix.
